zk.js/src/idls/psp_compressed_pda.ts

```diff
@@ -1121,6 +1180,41 @@ export const IDL: PspCompressedPda = {
         ]
       }
     },
+    {
+      "name": "CompressedProof",
+      "type": {
+        "kind": "struct",
+        "fields": [
+          {
+            "name": "proofA",
+            "type": {
+              "array": [
+                "u8",
+                32
+              ]
+            }
+          },
+          {
+            "name": "proofB",
+            "type": {
+              "array": [
+                "u8",
+                64
+              ]
+            }
+          },
+          {
+            "name": "proofC",
+            "type": {
+              "array": [
+                "u8",
+                32
+              ]
+            }
+          }
+        ]
+      }
+    },
```

Comment:
seems there's still `ProofCompressed` as well, which afaik has .a .b .c. 

can we remove one of them?

Reply:
We are moving away from verifier sdk. Changed the CompressedProof struct in compressed-pda utils.rs